libs/sdk-ui-dashboard/src/presentation/filterBar/configuration/ConfigurationModeSelect.tsx

```diff
@@ -0,0 +1,63 @@
+// (C) 2023 GoodData Corporation
+import React, { ChangeEvent, useCallback } from "react";
+import { IntlShape } from "react-intl/src/types.js";
+import { ModeSelectItem } from "./ConfigurationModeSelectItem.js";
```

Comment:
We need an empty line between lines 3 - 4 to separate the import libraries and import in the same project.